pages/requests/[uuid].js

```diff
@@ -117,7 +121,14 @@ const Request = () => {
           <CollapsibleSection header='Additional Information' description={request.htmlDescription} />
           <Title addClass='mt-4' title='Documents' size='small' />
           {documents.length ? documents.map((document, index) => (
-            <Document key={`${request.id}-${index}`} document={document} addClass='mt-3' />
+            <Document 
+              addClass='mt-3'
+              acceptSOW={acceptSOW}
+              accessToken={accessToken}
```

Comment:
why are we passing these into the component instead of handling them in the webstore? components should be primarily, and if possible exclusively, presentational.

- in this case, it should only care to display info and recognize that a button was clicked. once said button is clicked, that info can be passed back to the webstore and the appropriate actions taken. (e.g. pass back a string that triggers a switch statement)
- also, as we add more options to this drop down in the future, handling the result of them in the webstore becomes even more important.

Reply:
discussed & refactored slightly, when we have more actions in the dropdown we may refactor to use a switch case